chocolatey/choco-theme
# (#378) Update and add events

## Description Of Changes
This updates the Product Spotlight, Coffee Break, and Unpacking Software events. The right side flyout has been updated along with the add to calendar buttons.

## Motivation and Context
Events needed added and updated.

## Testing
1. [Merge this PR so images will load.](https://github.com/chocolatey/img/pull/7)
1. Run the website
2. Ensure home page features the new Unpacking Software event and add to calendar buttons work
3. The flyout has the new Unpacking Software event in it with the correct details
4. Clicking through to the event page works and shows the right information
5. The event listing page shows the correct events (new event added)
6. Once good, this PR (choco-theme) can be merged, but will need to be released and updated on the website before any additional PRs can be merged.

## Change Types Made

## Change Checklist


## Related Issue
* https://github.com/chocolatey/img/pull/7
* https://github.com/chocolatey/choco-theme/issues/378
* https://github.com/chocolatey/chocolatey.org/issues/300
* https://github.com/chocolatey/home/issues/295
* https://gitlab.com/chocolatey/community-infrastructure/community.chocolatey.org/-/issues/1245
